### ISSUE TITLE
Weather: Unable to get ForecastIO condition text in Fahrenheit

### ISSUE THREAD

**Issue by chuseman** [openhab/openhab1-addons#3062]
The [forecast.io API](https://developer.forecast.io/docs/v2) provides a nice daily summary:

```
Light rain on Sunday through Wednesday, with temperatures bottoming out at 84°F on Monday.
```

When the weather binding requests the forecast, it appends `units=si` to the query string, returning this instead:

```
Light rain on Sunday through Wednesday, with temperatures bottoming out at 29°C on Monday.
```

There doesn't appear to be a way to control this `units` parameter as it is hardcoded in `ForecastIoProvider`. I'm already appending `unit=fahrenheit` (and `mph`, `inches`, etc.) to all of my weather items to get the values in numbers that I understand. Unfortunately that won't help me here because this is a string that the forecast.io API publishes directly.

I'd like to suggest a way to control the setting of the units query string parameter. Perhaps a `LocationConfig` addition in the user's `openhab.cfg`:

```
weather:location.<locationId>.units=[auto|us|si|...]
```

No value here would still default to SI units but would be user configurable.

Would something like this be accepted? I'd be willing to spend some time implementing it.
##

**Comment by 9037568** [comment]
@chuseman care to try out a [test jar](https://github.com/openhab/openhab/files/668155/org.openhab.binding.weather-1.9.0-SNAPSHOT.zip) ?

The new config key:
````
#weather:location.<locationId1>.units=      (optional; defaults to "si")
````

**Comment by watou** [comment]
Closed via #5045.
